jetty-server/src/main/java/org/eclipse/jetty/server/Dispatcher.java

```diff
@@ -178,7 +178,7 @@ protected void forward(ServletRequest request, ServletResponse response, Dispatc
                 if (old_attr.getAttribute(FORWARD_REQUEST_URI) == null)
                     baseRequest.setAttributes(new ForwardAttributes(old_attr,
                         old_uri.getPath(),
-                        old_context == null ? null : old_context.getContextHandler().getContextPathEncoded(),
+                        old_context == null ? null : old_context.getContextHandler().getRequestContextPath(),
```

Comment:
I think this can just be 
```suggestion
                        baseRequest.getContextPath(),
```

which has some logic in it about cross context include dispatches.

Reply:
Fixed.